src/main/java/graphql/Assert.java

```diff
@@ -1,25 +1,26 @@
 package graphql;
```

Comment:
This is the KEY file changed and its test.

The others are all knock on effects